README.md

```diff
@@ -6,21 +6,17 @@
         <img src="https://fingerprintjs.github.io/home/resources/logo_dark.svg" alt="Fingerprint logo" width="312px" />
    </picture>
   </a>
+</p>
 <p align="center">
-<a href="https://github.com/fingerprintjs/fingerprint-pro-akamai-integration-property-rules">
-  <img src="https://img.shields.io/github/v/release/fingerprintjs/fingerprint-pro-akamai-integration-property-rules" alt="Current version">
-</a>
-<a href="https://opensource.org/licenses/MIT">
-  <img src="https://img.shields.io/:license-mit-blue.svg" alt="MIT license">
-</a>
-<a href="https://discord.gg/39EpE2neBg">
-  <img src="https://img.shields.io/discord/852099967190433792?style=logo&label=Discord&logo=Discord&logoColor=white" alt="Discord server">
-</a>
+<a href="https://github.com/fingerprintjs/fingerprint-pro-akamai-proxy-integration"><img src="https://img.shields.io/github/v/release/fingerprintjs/fingerprint-pro-akamai-integration-property-rules" alt="Current version"></a>
```

Comment:
This image needed to be updated too

Reply:
Sorry, done. Could you check again? @ilfa